multi-environments-kustomize/README.md

```diff
@@ -1,310 +1,31 @@
 #  Using ConfigSync with Multiple Environments
```

Comment:
I keep the README.md file here because it includes two tutorials, one for Config Sync Rendering, and the other for Config Build Rendering.

I removed the `Overview` section because it is included in the two tutorials.

Reply:
I understand now, thank you!